gadgets/bpfstats/gadget.yaml

```diff
@@ -0,0 +1,7 @@
+name: bpf stats
```

Comment:
I don't think it should be a gadget, but a separated command like in the PoC you implemented.

Reply:
> IMO, the fact that it's reusing the gadget lifecycle to run an "empty" gadget doesn't make it a gadget. It's an internal detail that doesn't need to be exposed to users.

I disagree with this. IMHO, the fact that it uses the lifecycle is exactly what makes it a gadget. It's similar to the discussion we had about mem/cpu usage. An annotation like the one used here should be used to enable internal datasources that emit events regarding mem/cpu usage. A basic gadget might just take that data and print it, while another gadget could combine that info with other data from ebpf/wasm/whatever.

Reply:
After some discussion we agreed it's better to have it as a gadget.